ex3/index.js

```diff
@@ -0,0 +1,24 @@
+const prompt = require("prompt-sync")({ sigint: true});
+
+const integerNumbers = [];
+for (let i = 0; i < 4; i++) {
+    console.log("Insert a integer number");
+    const input = prompt(">");
+    integerNumbers.push(Number(input));
+}
+
+function calculate() {
+    integerNumbers[0] = integerNumbers[0] + 25;
+    console.log(integerNumbers[0]);
+
+    integerNumbers[1] = integerNumbers[1] * 3;
+    console.log(integerNumbers[1]);
+
+    integerNumbers[2] = (integerNumbers[2] * 12) / 100;
+    console.log(integerNumbers[2]);
+    
+    integerNumbers[3] = integerNumbers[0] + integerNumbers[1] + integerNumbers[2]
+    console.log(integerNumbers[3]);
```

Comment:
This implementation doesn't meet the exercise requirements. The exercise specifies that you should "store in the fourth integer the sum of the original values (those entered by the user) of the first three integers." However, by mutating the original array during the calculations, you're using the modified values rather than the original ones. I recommend avoiding mutations and using new constants to preserve the initial values for the final sum calculation.

Reply:
I'm not sure I understand correctly, but this implementation is using the original values, not the modified ones.

Reply:
It isn't, please, take a closer look at your code. You are mutating the original array.

Reply:
The actual version might be working, as the 4th int is being evaluated before mutating the remaining ones, although I agree with Emilio that mutation user's inputs is not a good practice.
Please store the calculated numbers in new variables to avoid that.